src/UserCodeRunner.ts

```diff
@@ -390,9 +390,13 @@ export class UserCodeRuntimeError extends UserCodeError {
 
 	public get location(): { line: number; column: number } {
 		const stack = parse(this.error);
+		const userFileStackFrame = stack.find(callSite => callSite.getFileName() === USER_CODE_FILENAME);
+		if (userFileStackFrame === undefined) {
+			throw new Error('Runtime error detected outside of user code execution path. This is most likely a bug in the additional library source.');
```

Comment:
Is the original error message visible in this case, or does this `Error` obscure it?

Reply:
The original error would be obscured. Joel and I thought about it (and then I tested it) and the only way this could happen is if there is a raw throw in the additional source files.